docs/exts/exampleinclude.py

```diff
@@ -0,0 +1,182 @@
+from os import path
```

Comment:
Is this code copied from somewhere initially (looks like) or written from scratch? We need to have a licence header here and possibly link to where it was copied from.

Reply:
I copied the LiteralInclude directive from sphinx.  I know I have to add licenses, but this is WIP. I shared the code and the preview so that you would express opinions about the change.